### ISSUE TITLE
Consistency of (main) adapters overviews

### ISSUE THREAD

**Issue by boris-martin** [precice/precice.github.io#148]
Of the 7 "main" adapters, 4 of them have more than one page: CalculiX, SU2, OpenFOAM and deal.ii. However, each has a different structure. A common structure would be welcome.

Some spotted issues & fixes:
- Only deal.II describes the original solver. (SU2 in part) **=> Add "What is CalculiX?" and "What is OpenFOAM?"?** 
- History / references / cite section inconsistent, and empty with deal.ii (add the citation from the repo README : https://github.com/precice/dealii-adapter ?) **=> Make a References section with (if any) history + a "related literature" subsection?**
- "What is this" with OpenFOAM and SU2 => Split with "What is [solver]?" and "What is the adapter?" (Latter: plug-ing ? Standalone program? Different program)
- CalculiX has a "start here" section linking to other pages (build, configure, tutorials). Same for OpenFOAM and SU2 but it's called "Try".  **=> Rename CCX one to "Try"?** For deal.ii it's... Chaotic. Move things into a section that references new pages?

Suggested overall structure

- What is [solver name]?
- What does the adapter do/ what is it? (i.e. a plugin, a standalone app, a library, ...) **Unsure about the title**
- - Also include supported versions there?
- Usage
- - Links to get/build/configure/run tutorial pages
- - For deal.ii, also links to limitations, assumptions, theory, etc
- References / history / cite

Draft PR planned on my side but agreement on structure first would be better.

EDIT: some checkboxes:

- [ ]  Consistent introduction: 1) what kind of solver is this? 2) What kind of software is the adapter ? 
- [ ]  Is the next section (commong name? "Try" ? "Get started" ?) introducing links to build/configure the adapter.
- [ ]  Are there links to (main?) tutorials showcasing this adapter?
- [ ] Is the references section at the end consistent with others? The OpenFOAM one seems like a nice template. (Different font for references)

**Comment by MakisH** [comment]
In this context, we could also rename all overview pages to the same pattern and add redirections to their short form (e.g., from `openfoam-adapter.html` to `openfoam-adapter-overview.html`) #79.

Could you convert your summary to a comparison table or checkboxes? We could then assign sub-tasks to different people over the next weeks. I don't think we need a huge PR solving everything at once.

In short:
- I agree we should add a "what is [solver name]" everywhere.
- I agree we should add something about what the "adapter" actually means in the context of each solver.
- For the supported versions, we could even have a separate section. For the OpenFOAM adapter, this is even a separate page, due to complexity.
- For the "usage", we could even have an overview table or "badges"/"buttons" before the details.
- I am not sure if the deal.II part is chaotic, I would say it has quite a lot of nice documentation that is not really needed for other solvers. We could still define the basic sections/pages every adapter should have, but still allow for adapter-specific details.

**Comment by davidscn** [comment]
>  We could still define the basic sections/pages every adapter should have, but still allow for adapter-specific details.

I agree on that as our codes and adapters are quite different. We should use the overview pages in order to make it more clear where to find what.

**Comment by boris-martin** [comment]
Checkboxes added!
@MakisH the 4 overview pages currently have the `XX-adapter-overview.html` format.

**Comment by MakisH** [comment]
Thanks! If you want, feel free to push this forward with PRs. Otherwise, we could distribute the work during one of our next coding sessions.
